src/functions/Output.ps1

```diff
@@ -773,6 +829,69 @@ function Get-WriteScreenPlugin ($Verbosity) {
     New-PluginObject @p
 }
 
+function Format-CIErrorMessage {
+    [OutputType([System.Collections.Generic.List[string]])]
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [ValidateSet('AzureDevops', 'GithubActions', IgnoreCase)]
+        [string] $CIFormat,
+
+        [Parameter(Mandatory)]
+        [string] $Header,
+
+        [Parameter(Mandatory)]
+        [string[]] $Message
+    )
+
+    $CIOutputMapping = @{
+        'AzureDevops'   = @{
+            # header task issue error, so it gets reported to build log. https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=powershell#example-log-an-error
+            Header        = "##vso[task.logissue type=error] $Header"
+
+            # message error prefx, but doesn't get reported in build log. Only task issue errors get reported.
+            MessagePrefix = '##[error] '
+        }
+        'GithubActions' = @{
```

Comment:
Seems a bit hacky to format like this for GithubActions, but seems to align header with rest of message ok for now.